AdinoyiSadiq/MyDiary
# 159381385 Implement week1 feedback

### What does this PR do?
This pull request implements the feedback from the week1 review.

#### Description of Task to be completed?
1. Refactor authentication to use jsonwebtoken package
    - Install jsonwebtoken
    - Refactor code to authenticate a user using jsonwebtoken
2. Refactor api tests
3. Rename routes.js to index.js

#### How should this be manually tested?
- Setup the repo as instructed in the README
- run `npm run test` in the command line to see the tests pass

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#159381385](https://www.pivotaltracker.com/story/show/159381385)

#### Screenshots (if appropriate)
N/A

#### Questions:
None
